master/internal/rbac/api_rbac_stub.go

```diff
@@ -74,6 +74,12 @@ func (s *rbacAPIServerStub) RemoveAssignments(ctx context.Context,
 	return nil, UnimplementedError
 }
 
+func (s *rbacAPIServerStub) AssignMultipleGroups(ctx context.Context, req *apiv1.AssignMultipleGroupsRequest) (
```

Comment:
blocking: this endpoint can just live in OSS in ```internal/usergroup```

so you should be able to avoid the ee pain

since usergroup stuff is technically OSS but not exposed

Reply:
alright, current status is, I moved it but I would need to add tests (for example, adding group-user pairs in a batch is probably going to error out if one of those users is already in the group)